### PR TITLE
FMODAudioEngine::stopAllMusic

### PR DESCRIPTION
**Pull request by hiimjustin000**
this took a toll on me

Note: I need someone with at least some FMOD experience to review my decompilation of the function, as `Channel::stop` doesn't exist in FMOD, but `ChannelControl::stop` does. Should I change the member to a `ChannelControl`, or leave it as a `Channel`?

### CODE REVIEW

File: bindings/2.206/GeometryDash.bro [hunk 1]
```diff
@@ -4194,10 +4199,10 @@ class FMODAudioEngine : cocos2d::CCNode {
 
 	virtual void update(float) = win 0x54510, m1 0x362540, imac 0x3e2ac0;
 
-	PAD = win 0xc4, android32 0x54, android64 0xac, imac 0x7c, m1 0x7c;
+	PAD = win 0xc4, android32 0x54, android64 0xac, mac 0x7c;
```
Comment (Fleeym) [inline bindings/2.206/GeometryDash.bro:4202]: Why did you change this from imac, m1 to mac?

**Comment by matcool** [comment]
> this took a toll on me
> 
> Note: I need someone with at least some FMOD experience to review my decompilation of the function, as `Channel::stop` doesn't exist in FMOD, but `ChannelControl::stop` does. Should I change the member to a `ChannelControl`, or leave it as a `Channel`?

Channel inherits ChannelGroup. no need to change anything
